README.md

```diff
@@ -31,7 +36,7 @@ On a local machine:
 3. Set the location of the weather database in the Main.R script of the R_program folder (where it says set database location), along with the name of the weather database (where it says Provide the name of the database in quotes).
 4. Add site ids, you wish to run the wrapper on, to the siteid variable (third line from top) in the generate_rSFSTEP2_structure.sh script. Site 1 and 2 are present as examples.
 5. Run the cloneSTEPWAT2.sh script.
-6. Run the generate_rSFSTEP2_structure.sh script. The parameters are <Location of R_program> <number_of_sites> <number_of_scenario>
+6. Run the generate_rSFSTEP2_structure.sh script. The parameters are <R_program> <number_of_sites> <number_of_scenario>
 7. Run the run_local.sh script.
```

Comment:
I guess that this could be clarified to be 
```
Run the call_sbatch.sh script. The parameter is <number_of_sites>
```
based on errors I get with `./call_run_local.sh` and the code comments.

Reply:
addressed by commit https://github.com/DrylandEcology/rSFSTEP2/commit/b1324ec6e41fbada4567efb09acd29e6e3387079